bin/fuel-convert

```diff
@@ -10,6 +10,18 @@ from fuel.converters.base import MissingInputFiles
 from fuel.datasets import H5PYDataset
 
 
+class SplitOutputAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        absolute_path = os.path.abspath(values)
+        if os.path.isdir(absolute_path):
```

Comment:
I'm a bit confused by how things would work after this PR. If the user overwrites the filename, that means the corresponding `Dataset` class won't be able to find the file at all, right?

@rizar Nope, a path to a directory doesn't need to end with a `/`, and even if you would require that, it depends on the operating system (e.g. `\` in Windows).

Can't we split this into two flags? One for the output directory, one for the output file name. That avoids the need for tricky argument parsing, and makes it easier to give a warning about using that option (e.g. that the `Dataset` might not be able to load things automatically anymore).